lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -205,6 +206,14 @@ def __init_pulsar_app( self, pulsar_conf_path ):
             log.info("Loading Pulsar app configuration from %s" % pulsar_conf_path)
             with open(pulsar_conf_path, "r") as f:
                 conf.update(yaml.load(f) or {})
+        if "job_metrics_config_file" not in conf:
+            conf["job_metrics"] = self.app.job_metrics
```

Comment:
``job_metrics_config_file`` in ``conf`` would mean the deployer specified a file explicitly and so that should be respected. Hence ``if "job_metrics_config_file" not in conf`` then a file wasn't specified and the ``conf`` object can just be updated to inject Galaxy's ``job_metrics`` object directly. I think this correct.